### PR TITLE
Create MajorNormalization.md

### PR DESCRIPTION
**Pull request by matthuhman**

### CODE REVIEW

File: MajorNormalization.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+```
+
+
+#Response Information
+
+The response returns a single data node which contains a list of normalized majors. These normalized majors are ordered by the confidence score. Confidence scores range from 0.0 to 1.0. 
+
+#Versioning
+-----------
+The response from the School Normalization Service is versioned with the current version being 1.0. The data set used to perform the normalization is unversioned.
```
Comment (andhesmi) [inline MajorNormalization.md]: The response from the Major Normalization...

Comment (matthuhman) [reply MajorNormalization.md]: missed a school, whoops

File: MajorNormalization.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- [Versioning](#versioning)
+
+
+
+Request Information
+-----
+
+HTTP method: GET or POST
+Parameters (query/form):
+-       major_name (required) : The major name to be normalized
```
Comment (andhesmi) [inline MajorNormalization.md]: - bullet point

**Comment by andhesmi** [comment]
:+1: even though I don't approve the table with a box

File: MajorNormalization.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- [Sample Response](#sample-response)
+- [Versioning](#versioning)
+
+
+
+Request Information
+-----
+
+HTTP method: GET or POST
+Parameters (query/form):
```
Comment (thomasmahoney) [inline MajorNormalization.md:16]: did you preview this? this doesn't render as a new line.

File: MajorNormalization.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+
+Response Information
+-----
+
+The response returns a single data node which contains a list of normalized majors. These normalized majors are ordered by the confidence score. Confidence scores range from 0.0 to 1.0. 
+
+Versioning
+-----------
+The response from the Major Normalization Service is versioned with the current version being 1.0. The data set used to perform the normalization is unversioned.
```
Comment (thomasmahoney) [inline MajorNormalization.md:62]: > The data set used to perform the normalization is unversioned.

are you sure about that? SNorm and CNorm both use versioned data sets. I'd be surprised if MNorm were different.

File: MajorNormalization.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+-----
+
+```
+{
+    "data": {
+        "normalized_majors": [
+            {
+                "normalized_name": "Electrical Engineering",
+                "cip_code": 15.001,
+                "confidence": 1.0,
```
Comment (thomasmahoney) [inline MajorNormalization.md:47]: this is syntactically incorrect due to the trailing comma here

File: MajorNormalization.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+            }
+        ]
```
Comment (thomasmahoney) [inline MajorNormalization.md:49]: we should have a data_version node like we do for the other two normalization services

**Pull request merged**